pkg/porter/create_test.go

```diff
@@ -39,5 +39,37 @@ func TestCreate(t *testing.T) {
 	dockerignoreStats, err := p.FileSystem.Stat(".dockerignore")
 	require.NoError(t, err)
 	tests.AssertFilePermissionsEqual(t, ".dockerignore", pkg.FileModeWritable, dockerignoreStats.Mode())
+}
 
+func TestCreateInDirectory(t *testing.T) {
+	p := NewTestPorter(t)
+	defer p.Close()
+
+	err := p.Create("mybundle")
```

Comment:
If we pass in `mybundle/yourbundle/superbundle` will this work? 

Helm has a thing where it will create the directory (base directory) if the others exists.  So it `os.Stat`'s the previous directories before making a `helm create` in the last _(superbundle)_ "directory" assuming the last isn't a file.

We are creating the whole path regardless if it's a file at that path to include a directory of that and the bundle name will be `mybundle/yourbundle/superbundle`.  

We may need to Stat the filepath of the directories if they are passed like this to ensure we do not have a bundle name with slashes, etc.  

Helm has an example of this in their create flow.

Reply:
Oh I am sorry I was a bit busy at the moment to answer. Let me get back on to this 
>If we pass in mybundle/yourbundle/superbundle will this work?
Yes this will work. But there's a catch. It would update the directory if it exists. Now I want to ask should we block the user from running porter create or porter create mybundle if any of the related files exist or should we stick with the behavior that it is in.

Reply:
> Yes this will work. 

I'm curious as what the name will be. Will it be `mybundle/yourbundle/superbundle`

> But there's a catch. It would update the directory if it exists. 

I don't think that's a problem.  Helm does the same thing.  If there exists a directory, it will overwrite what's in there if the target exists. 

> Now I want to ask should we block the user from running porter create or porter create mybundle if any of the related files exist or should we stick with the behavior that it is in.

The existing functionality will allow you to create a bundle in the current directory if anything exits  there and overwrite what is laid down on disks as an overwritten file.  This behavior should stay and expect that to be the case with the target directory _(if new created, if old overwrite)_

Reply:
> I'm curious as what the name will be. Will it be mybundle/yourbundle/superbundle
it will be named superbundle. And will be inside youbundle folder which will be inside, superbundle. It would work exactly like how helm create works
>The existing functionality will allow you to create a bundle in the current directory if anything exits there and overwrite what is laid down on disks as an overwritten file. This behavior should stay and expect that to be the case with the target directory (if new created, if old overwrite)
Sure I have made those small changes will push it in few moments. Let me know if anything else is required.

Reply:
@troy0820 now it is 100% similar to helm create. I have also written the test cases to prove the point and looked at them locally as well. Kindly take a look.